about.html

```diff
@@ -0,0 +1,105 @@
+            <h2 class="bigtitle">
+                STAND UP COMEDY <br>
+                INSTAGRAM CELEBRITIES <br>
+                2023
+            </h1>
+            <div class="eventdescription2">
+                In this event, you will recover the sense ofhumor that 
+                was lost inside of you, making you feel alive and more 
+                sociable with others.
+                <br>
+                This activity aims for people above 18 years 
+                because of the bad words or the content used that is 
+                not recommended for kids.
+            </div>
+            <h3></h3>
+            <p>
+                Feel free to contact us by Email for any question about this stand up comedy event!
+            </p>
+            <p class="myemail pb128">
+                gabrielrozas1996@gmail.com
+            </p>
+        </section>
+        <section class="logoselection dflex flexcolumn justifyandaligncenter py64">
+            <h3 class="textcenter py16">Stand up comedy instagram celebrities logo</h3>
+            <span class="spanline1"></span>
+            <p class="textcenter py16">This logo was selected from a free copyright webpage <a href="https://www.iconpacks.net/" target="_blank" rel="noopener">https://www.iconpacks.net/</a></p>
+            <div class="imgcontainer mt32"><img class="logo" src="img/microphone-14646.png" alt="logo"></div>
+        </section>
+        <section class="logoselection dflex flexcolumn justifyandaligncenter py64">
+            <span class="spanline2 my16"></span>
+            <h3 class="textcenter py16">See past Stand up comedy instagram celebrities</h3>
+            <span class="spanline1"></span>
+            <p class="textcenter py16">See past events featuring some of the comedians invited for this event too!</p>
+            <div class="pasteventscontainer dflex flexcolumn spacebetween">
+                <img class="pastevent py16" src="img/pasteventluke.png" alt="eventimg">
+                <img class="pastevent py16" src="img/pasteventmatt.png" alt="eventimg">
+            </div>
+        </section>
+
+        <section class="partners dflex flexcolumn justifyandaligncenter">
+            <h3 class="py16">Partners</h3>
+            <span class="spanline1"></span>
+            <p class="py16 dflex fwrap justifyandaligncenter"><img src="img/partner1.png" alt="partnerslogos"> <img src="img/partner2.png" alt="partnerslogos"> <img src="img/partner3.png" alt="partnerslogos"> <img src="img/partner4.png" alt="partnerslogos"> <img src="img/partner5.png" alt="partnerslogos"></p>
+        </section>
+        <section class="endpage dflex">
+            <h2 class="py64">
+                Gabo Productions Inc.
+            </h2>
+            <p class="py16 py64">
+                2023 Gabo productions Inc. <br>
+                Some Right Reserved
+            </p>
+        </section>
+        <script src="aboutfeatures.js"> </script>
```

Comment:
Great job @EmiLoKiJu 🤝

- [ ] [Optional] Kindly ensure your desktop About section footer matches the one given in the design given.
- Your design
![your-about-footer](https://user-images.githubusercontent.com/63917725/233514265-22190cfd-2b08-4cb0-906e-3642e5b32a4c.PNG)
- Given design
![given-about-footer](https://user-images.githubusercontent.com/63917725/233514302-ee68c7ea-8104-42b9-8be8-ac2048050d50.PNG)